src/rest_api/formatters/object.clj

```diff
@@ -333,43 +333,105 @@
              (:evidence/go-term-evidence holder)))
 
    :Expr_pattern_evidence
-   (if-let [epe (:evidence/expr-pattern-evidence holder)]
+   (when-let [epe (:evidence/expr-pattern-evidence holder)]
      (map (partial pack-obj "expr-pattern") epe))
 
    :Microarray_results_evidence
-   (if-let [e (:evidence/microarray-results-evidence holder)]
+   (when-let [e (:evidence/microarray-results-evidence holder)]
      (map (partial pack-obj "microarray-results") e))
 
    :RNAi_evidence   ;; could be multiples?
-   (if-let [rnai (first (:evidence/rnai-evidence holder))]
+   (when-let [rnai (first (:evidence/rnai-evidence holder))]
      {:id (:rnai/id rnai)
       :label (if-let [hn (:rnai/history-name rnai)]
                (format "%s (%s)" (:rnai/id rnai) hn)
                (:rnai/id rnai))})
 
    :Feature_evidence
-   (if-let [features (:evidence/feature-evidence holder)]
+   (when-let [features (:evidence/feature-evidence holder)]
      (map (partial pack-obj "feature") features))
 
    :Laboratory_evidence
-   (if-let [labs (:evidence/laboratory-evidence holder)]
+   (when-let [labs (:evidence/laboratory-evidence holder)]
      (map (partial pack-obj "laboratory") labs))
 
    :From_analysis
-   (if-let [anas (:evidence/from-analysis holder)]
+   (when-let [anas (:evidence/from-analysis holder)]
      (map (partial pack-obj "analysis") anas))
 
    :Variation_evidence
-   (if-let [vars (:evidence/variation-evidence holder)]
+   (when-let [vars (:evidence/variation-evidence holder)]
      (map (partial pack-obj "variation") vars))
 
    :Mass_spec_evidence
-   (if-let [msps (:evidence/mass-spec-evidence holder)]
+   (when-let [msps (:evidence/mass-spec-evidence holder)]
      (map (partial pack-obj "mass-spec-peptide") msps))
 
    :Sequence_evidence
-   (if-let [seqs (:evidence/sequence-evidence holder)]
-     (map (partial pack-obj "sequence") seqs))))
+   (when-let [seqs (:evidence/sequence-evidence holder)]
+     (map (partial pack-obj "sequence") seqs))
+
+   :genotype
+   (when-let [c (:anatomy-function.assay/condition holder)]
+     (first (:condition/genotype c)))
+
+   :Autonomous
+   (when-let [ss (:anatomy-function-info/autonomous holder)]
+     (for [s ss]
+       {:taxonomy "all"
+        :class "txt"
```

Comment:
Might need to double check if the class should 'txt' or 'text'

Reply:
This is how the object are coming from Catalyst API. I am leaving it the same for now. In the future we could look into making this more consistent.